ci/gh-install.sh

```diff
@@ -37,42 +35,99 @@ case "$ID" in
         # be answered.
         export DEBIAN_FRONTEND=noninteractive
 
+        # Debian upstream data:
+        # https://tracker.debian.org/pkg/ostree
+        # https://salsa.debian.org/debian/ostree
+        # https://salsa.debian.org/debian/ostree/-/blob/debian/master/debian/control
+        #
+        # Ubuntu package data:
+        # https://packages.ubuntu.com/source/impish/ostree
+        #
         # TODO: fetch this list from the Debian packaging git repository?
+
+        # First construct a list of Build-Depends common to all
+        # versions. This includes build-essential, which is assumed to
+        # be installed on all Debian builders. We also add gjs to allow
+        # the JS tests to run even though gjs is explicitly disable in
```

Comment:
Oh, thanks for the reminder, I should put that back after bullseye gets released.

Reply:
Yep, we'll stay on target there.